cmd/updatetree/main.go

```diff
@@ -114,5 +129,9 @@ func main() {
 	// DO NOT change the output format, scripts are meant to depend on it.
 	// If you really want to change it, provide an output_format flag and
 	// keep the default as-is.
```

Comment:
Comment could do with updating and/or moving into the `else` arm maybe?

Reply:
Done.